### PR TITLE
jQuery UI: Article on the classes option

### PR DESCRIPTION
**Pull request by scottgonzalez**
This is an update to #574 based on the feedback on that PR.

### CODE REVIEW

**Comment by arschmitz** [comment]
Over all this looks good to me. Wondering if it's worth adding examples of retrieving current value and setting a value ( other then `null` ) after init. These are not any different then other widgets but just for the sake of completeness of the article.

I'm fine either way just throwing this out there.

File: page/jquery-ui/widget-factory/classes-option.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+The second thing to note is the use of the widget factory's new `_addClass()` and `_removeClass()` methods. The methods are shorthands that accept a structural class name and apply it, and its associated presentational class names, to an element. You can specify the element by passing it as the first argument of `_addClass()`/`_removeClass()` (or `_toggleClass()`, which hasn't been mentioned yet, but it also available), or you can omit the element as this example does, in which case the classes are applied to or removed from the widget's main element (`this.element`). For more examples of how these methods work, refer to the API documentation of [`_addClass()`]((http://api.jqueryui.com/jquery.widget/#method-_addClass), [`_removeClass()`]((http://api.jqueryui.com/jquery.widget/#method-_removeClass), and [`_toggleClass()`]((http://api.jqueryui.com/jquery.widget/#method-_toggleClass).
```
Comment (agcolom) [inline page/jquery-ui/widget-factory/classes-option.md]: small typo: but it also available -> but is also available

Comment (scottgonzalez) [inline page/jquery-ui/widget-factory/classes-option.md]: This doesn't exist in the article. I removed this whole section.

File: page/jquery-ui/widget-factory/classes-option.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+In general, using the `classes` options in your widgets gives your widgets' users flexibility in how your widgets' class names are managed, and lets them adapt to a wide variety of themes.
```
Comment (agcolom) [inline page/jquery-ui/widget-factory/classes-option.md]: options in your widgets gives .. and lets them -> options in your widgets **give** .. and **let** them 
OR 
**option** in your widgets gives .. and lets them

**Comment by agcolom** [comment]
Sorry for the comments on text that has now been removed!

**Comment by scottgonzalez** [comment]
I merged as is, since @arschmitz approved. We can always add more, but I want to get our release announced ASAP.

**Comment by arschmitz** [comment]
👍 💯

**Pull request closed without merging**